src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -68,9 +68,10 @@ export class TableInteractorColumnComponent {
     if (this.navigatorComponents()[i].expanded) {
       this.collapseAllButOne(i);
       this.hideRowsDisplayedAsSubcomponents(i);
-    } else {
-      this.displayAllRows();
     }
+    // else {
```

Comment:
What's the bug?

Reply:
I mentioned you on the issue Eliot created